docs/administration/troubleshooting-published-data-packages.md

```diff
@@ -0,0 +1,91 @@
+---
+parent: Administration Tasks
+title: Troubleshooting published data packages
+nav_order: 21
+---
+# How to troubleshoot published data packages
+
+This documentation explains how users can diagnose issues with all existing published data packages using our new troubleshooting tool. The tool performs a series of checks to verify the integrity and compliance of data packages. The scope of this tool includes both data packages that have been successfully published and those that have failed to publish (packages that initiated the publication process but did not succeed). Specifically, it targets data packages with their Attribute-Value Units (AVUs) including `org_publication_status` of `OK`, `Retry`, `Unrecoverable`, or `Unknown`. Note, the `org_` prefix may vary in different Yoda instance as it is a constant variable of `UUORGMETADATAPREFIX` stored in `constant.py`
+
+Alternatively, the tool can diagnose a specific data package when provided with its name.
+
+**Requirement:** Python 3 or higher required, Yoda version 1.10 or later, rodsadmin user
+
+## Check Steps
```

Comment:
I added the Check Steps here, but i was hesitating this can be a part of text in design section not the administration section. In the end, I thought this can help administrator understands the check steps so I put it here. Other remarks / suggestions are welcomed